python/lsst/ctrl/mpexec/mpGraphExecutor.py

```diff
@@ -70,12 +70,18 @@ def __init__(self, qnode):
         self._state = JobState.PENDING
         self.started = None
         self._rcv_conn = None
+        self._terminated = False
 
     @property
     def state(self):
         """Job processing state (JobState)"""
         return self._state
 
+    @property
+    def terminated(self):
+        """Return True if job was killed by stop() method (`bool`)"""
```

Comment:
but also if the exit code was negative?

Reply:
This is sort of implied, if `stop` was called but did not kill the process then it was not killed by stop. I'll expand the docstring.